dev_settings.py

```diff
@@ -10,11 +10,11 @@
 LOCAL_APPS = (
     'django_extensions',
     'kombu.transport.django',
-    # for tests
-    'testapps.test_elasticsearch',
-    'testapps.test_pillowtop',
 )
 
+# TEST_RUNNER is overridden in testsettings, which is the default settings
+# module for the test command (see manage.py); this has no effect by default.
+# Use ./manage.py test --settings=settings to use this setting.
 TEST_RUNNER = 'testrunner.DevTestRunner'
```

Comment:
Very good question. I'm working on a [branch to remove old/unused test code](https://github.com/dimagi/commcare-hq/compare/cleanup-tests), and I was not able to immediately able to determine if it's still needed. Maybe it's still used for JS tests?